cli/src/pcluster/schemas/imagebuilder_schema.py

```diff
@@ -0,0 +1,128 @@
+from marshmallow import fields, post_load
+
+from pcluster.config.imagebuilder_config import (
+    ImageBuilderBucketLogConfig,
+    ImageBuilderBuildConfig,
+    ImageBuilderChefCookbookConfig,
+    ImageBuilderComponentsConfig,
+    ImageBuilderConfig,
+    ImageBuilderDevSettingsConfig,
+    ImageBuilderImageConfig,
+    ImageBuilderVolumeConfig,
+)
+from pcluster.schemas.cluster_schema import BaseSchema
+
+
+class ImageBuilderVolumeSchema(BaseSchema):
+    """Represent the schema of the ImageBuilder Volume."""
+
+    size = fields.Int(data_key="Size")
+    delete_on_termination = fields.Bool(data_key="DeleteOnTermination")
+    encrypted = fields.Bool(data_key="Encrypted")
+    kms_key_id = fields.Str(data_key="KmsKeyId")
+
+    @post_load()
+    def make_config(self, data, **kwargs):
+        """Generate config object."""
+        return ImageBuilderVolumeConfig(**data)
+
+
+class ImageBuilderComponentsSchema(BaseSchema):
+    """Represent the schema of the ImageBuilder components."""
+
+    component_arns = fields.List(fields.Str, data_key="ComponentArns", required=True)
```

Comment:
Once the @hanwen-pcluste patch will be merged you can avoid to specify the data_key for all the attributes that are aligned to the configuration file attribute and the only difference is CamelCase vs snake_case